msi/GitProduct.wxs

```diff
@@ -158,6 +167,11 @@
       <Directory Id='ProgramFiles$(var.SixtyFourBit)Folder' Name='PFiles'>
         <Directory Id='INSTALLFOLDER' Name='Git'>
           <Directory Id='BinFolder' Name='bin' />
+        <?if $(var.SixtyFourBit)=64 ?>
+          <Directory Id='MingwFolder' Name='mingw64' />
+        <?else?>
+          <Directory Id='MingwFolder' Name='mingw32' />
+        <?endif?>
```

Comment:
I would like to get all conditionals that can live in either `release.sh` or a `.wxs` file back into the `release.sh`. If some conditional *must* live in the WiX files, so be it, but if it does not, let's keep the `.wxs` files as conditional-free as possible.